blog/README.md

```diff
@@ -67,6 +67,11 @@ The code is statically analyzed with [Psalm](https://psalm.dev/). To run static
 ./vendor/bin/psalm
 ```
 
+### Installing Bootstrap Node Modules with WAMP 
+
+1. Make a directory node_modules in the blog directory.
```

Comment:
Isn't `npm` creating `node_modules` automatically?

Reply:
**Testing Demo Installation:**
Step 1: Downloaded zip and extracted into demo2 folder
Step 2: composer update
Step 3: c:\wamp64\www\demo2\blog>yii serve

![image](https://github.com/yiisoft/demo/assets/8538339/6df7c117-69e3-41b0-9f4f-d541f1a6834f)

Step 4: Whilst running wampserver in url typed: 127.0.0.1:8080
![image](https://github.com/yiisoft/demo/assets/8538339/77234e52-df1b-49cd-8a76-c3535ceed579)

Step 5: Removed blog from .env file path to remove above error.
![image](https://github.com/yiisoft/demo/assets/8538339/65ba0719-0194-4c13-ae0c-bd84e21d5a18)

Reply:
![image](https://github.com/yiisoft/demo/assets/8538339/d369720a-34dd-44bf-867a-4945162dbb1d)